app/src/main/res/values/strings.xml

```diff
@@ -876,6 +876,8 @@
     <string name="media_gallery_on_image_download_error">There was an error when saving the image to
         Downloads folder. Please check your Internet connection and try again
     </string>
+    <string name="certificate_download_error">There was an error when saving the certificate to
+        Downloads folder. Please check your Internet connection and try again</string>
```

Comment:
Maybe we can change this string? As thinking a bit more on it, its used for when user denied permissions to write into external storage, any ideas welcome

Reply:
for the copy please ask @Astrid-Wire